Copy editing: timeseries (#10147)
astropy/astropy: docs/timeseries/lombscargle.rst

Lomb-Scargle Periodograms
*************************

> The Lomb-Scargle Periodogram (after Lomb [1]_, and Scargle [2]_) is a commonly

Reviewer: periodogram wasn't capitalized mid sentence in the BLS docs above and I think it shouldn't be in this page either

used statistical tool designed to detect periodic signals in unevenly spaced
observations. The :class:`~astropy.timeseries.LombScargle` class is a unified
interface to several implementations of the Lomb-Scargle Periodogram, including
a fast *O[NlogN]* implementation following the algorithm presented by Press &
Rybicki [3]_.

The code here is adapted from the `astroml`_ package ([4]_, [5]_) and the
`gatspy`_ package ([6]_, [7]_).  For a detailed practical discussion of the
Lomb-Scargle Periodogram, with code examples based on ``astropy``, see
*Understanding the Lomb-Scargle Periodogram* [11]_, with associated code at
https://github.com/jakevdp/PracticalLombScargle/.

.. _gatspy: http://www.astroml.org/gatspy/
.. _astroml: http://astroml.org/

Basic Usage
===========

.. Note::
   All frequencies in :class:`~astropy.timeseries.LombScargle` are **not**
   angular frequencies, but rather frequencies of oscillation (i.e., number of

>    cycles per unit time.)

Reviewer: ```suggestion
   cycles per unit time).
```

The Lomb-Scargle Periodogram is designed to detect periodic signals in
unevenly spaced observations.

Example
-------

> .. EXAMPLE START: Using the LombScargle Class to Detect Periodic Signals

Reviewer: ```suggestion
.. EXAMPLE START: Using the Lomb-Scargle Periodogram to Detect Periodic Signals
```

The periodogram shows a clear spike at a frequency of 1 cycle per unit time,
as we would expect from the data we constructed.

.. EXAMPLE END

Measurement Uncertainties
-------------------------

The :class:`~astropy.timeseries.LombScargle` interface can also handle data with
measurement uncertainties.

Example
^^^^^^^

> .. EXAMPLE START: Using the LombScargle Class for Measurement Uncertainties

Reviewer: ```suggestion
.. EXAMPLE START: Using the Lomb-Scargle Periodogram with Measurement Uncertainties
```

For a detailed discussion of these caveats and others when computing and
interpreting false alarm probabilities, please refer to [11]_.

Periodogram Algorithms
======================

The :class:`~astropy.timeseries.LombScargle` class makes available
several complementary implementations of the Lomb-Scargle Periodogram,
which can be selected using the ``method`` keyword of the Lomb-Scargle power.
By design all methods will return the same results (some approximate),
and each has its advantages and disadvantages.

> For example, to compute a periodogram using the Fast Chi-squared method

Reviewer: I don't think this needs the capitalization

Author: It's capitalized on the arxiv: https://arxiv.org/abs/0901.1913 but is it common enough that people generally use it less formally?

Reviewer: Interesting. I would have assumed it is the same  logic as "fast" in the fast Fourier transform, which is capitalized in the abbreviation but not in full text form, that is also along the line that`chi-squared` is usually kept lower case, or is written with the Greek letter

Author: Looks like Fast Fourier Transform is also all caps in that paper, so maybe they just like capitalizing!
